.github/workflows/release.yml

```diff
@@ -1,9 +1,36 @@
-name: Publish JS gRPC bindings
+name: Release
```

Comment:
No reason to have multiple GH Action files to do things we want done on release. Consolidated here.